src/dialogs/device-registry-detail/dialog-device-registry-detail.ts

```diff
@@ -163,7 +171,11 @@ class DialogDeviceRegistryDetail extends LitElement {
       });
       this._params = undefined;
     } catch (err) {
-      this._error = err.message || "Unknown error";
+      this._error =
+        err.message ||
+        this.hass.localize(
+          "ui.panel.config.entity_registry.editor.unknown_error"
```

Comment:
```suggestion
          "ui.panel.config.devices.unknown_error"
```

Reply:
Implemented in https://github.com/home-assistant/home-assistant-polymer/pull/4141/commits/96f6a843a110bbac601076e246129eb7376adc0b